codalab/rest/bundles.py

```diff
@@ -105,6 +105,8 @@ def _fetch_bundles():
     specs = query_get_list('specs')
     worksheet_uuid = request.query.get('worksheet')
     descendant_depth = query_get_type(int, 'depth', None)
+    command = query_get_type(str, 'command', '')
```

Comment:
We should document the following parameters too.

Reply:
Done